### PR TITLE
Smoke test the new retirement spanish url

### PR DESCRIPTION
**Pull request by wpears**
Missed this in #8425

### CODE REVIEW

**Review by chosak — DISMISSED** [review]

File: cfgov/scripts/http_smoke_test.py [hunk 1]
```diff
@@ -119,7 +119,7 @@
     "/paying-for-college/",
     "/paying-for-college2/understanding-your-financial-aid-offer/about-this-tool/",  # noqa: E501
     "/retirement/before-you-claim/",
-    "/retirement/before-you-claim/es/",
+    "/es/retirement/before-you-claim/",
```
Comment (chosak) [inline cfgov/scripts/http_smoke_test.py:122]: Shouldn't this be /es/herramientas-del-consumidor/jubilacion/antes-de-solicitar/? Why do both https://www.consumerfinance.gov/consumer-tools/retirement/before-you-claim/es/ and https://www.consumerfinance.gov/es/herramientas-del-consumidor/jubilacion/antes-de-solicitar/ still work?

Comment (wpears) [reply cfgov/scripts/http_smoke_test.py]: Fixed in d35096f (plus a bonus fix to a url I missed)

**Comment by chosak** [comment]
Something seems wrong. If I visit http://localhost:8000/es/herramientas-del-consumidor/jubilacion/antes-de-solicitar/, and try the app, it tries to hit the URL /es/herramientas-del-consumidor/retirement-api/estimator/1-1-1970/100000/es/, but that returns a 404. Is there a need to define [this alternative Spanish API endpoint](https://github.com/cfpb/consumerfinance.gov/blob/74411e65ac84c64b2319cd44e0e69c0d3c2111dc/cfgov/retirement_api/es_urls.py#L27-L32) at all? There's nothing in the API endpoint (http://localhost:8000/consumer-tools/retirement/retirement-api/estimator/1-1-1970/100000/) that's language-dependent. Couldn't the Spanish version of the pages call the (English) API?

**Comment by wpears** [comment]
You'll need to build the js with `yarn run build` (that endpoint is constructed on the fly, or used to be at least).

I'll check out the second part, you may be right.

**Review by chosak — APPROVED** [review]
Thanks, works for me. It would be nice to have some Spanish Cypress tests [here](https://github.com/cfpb/consumerfinance.gov/blob/74411e65ac84c64b2319cd44e0e69c0d3c2111dc/test/cypress/integration/consumer-tools/before-you-claim/before-you-claim.cy.js#L52) to make sure this behavior continues to work.

**Pull request merged**